zign/api.py

```diff
@@ -66,6 +66,11 @@
             }}
         }})();
     </script>
+    <noscript>
```

Comment:
I think it's a good idea to do this, however:

* Multiple nodes with same ID (`error`) in this document => undefined behavior. Don't do it.
* `noscript` in `head` is not allowed to contain flow content such as `<p>`, put it in `<body>`.
* What's the point of `display:none`? `noscript` content will only be inserted when scripting is disabled.

Suggestion:

~~~ html
<body>
  <noscript>
    <p>Your browser does not support Javascript! Please enable it or switch to a Javascript enabled browser.</p>
  </noscript>
  <!-- actual content -->
</body>
~~~